src/TodoBox.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import Todo from './Todo';
+import Button from './Button';
+
+function TodoBox({ todo, onClick }) {
+  return (
+    <li>
+      <Todo todo={todo} />
```

Comment:
지금 복잡도에서는 굳이 컴포넌트로 추출하지 않아도 될것같네요 :)
그리고 이 경우에는
```suggestion
      <Todo content={todo.content} />
```
이렇게 넘기는게 좋겠네요